src/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Components.css';
```

Comment:
I like that you're using css this way, but try to use css that only applies to this file?  Otherwise, use it in the index...